spec.md

```diff
@@ -246,8 +249,9 @@ Policies MUST specify the following fields:
   clients SHOULD cache a policy for up to this value from last policy fetch
   time.
 * rua: Address to which aggregate feedback MAY be sent (comma-separated
-  plain-text list of email addresses, optional). For example,
-  "mailto:postmaster@example.com" from [@!RFC3986].
+  plain-text list of email addresses or an HTTPS endpoint, optional). For example,
+  "mailto:postmaster@example.com" from [@!RFC3986] or HTTP POST to "https://example.com/sts-report".
```

Comment:
I'm moving the thing about POST'ing as a note in the "Failure Reporting" section.